wiki/Configuration:-Key-Bindings.md

```diff
@@ -304,15 +305,17 @@ binds {
 }
 ```
 
-#### `screenshot-screen`, `screenshot-window`
+#### `screenshot`, screenshot-screen`,`screenshot-window`
```

Comment:
I feel that it's better to have a separate `screenshot` section. This is conflating them a bit too much.

Reply:
Agreed. I'll see to that now.

Reply:
Would you like me to make a screenshot section within the keybinding docs, or within miscellaneous?

Reply:
I think don't add new sections, just add another "#### `screenshot`"